mono/eglib/glib.h

```diff
@@ -1123,6 +1123,12 @@ gboolean   g_file_test (const gchar *filename, GFileTest test);
 #else
 #define g_write write
 #endif
+#ifdef G_OS_WIN32
+#define g_read g_assert_not_reached
+#else
+#define g_read read
+#endif
```

Comment:
Is this needed?

Reply:
I'll make it _read, I just haven't looked into Windows yet